orga/app/components/routes/authenticated/sup-students/list-items.js

```diff
@@ -1,14 +1,30 @@
 import { inject as service } from '@ember/service';
 import Component from '@ember/component';
```

Comment:
```suggestion
import Component from '@glimmer/component';
```

glimmeriser le fichier